common-pg/src/main/scala/com/socrata/pg/analyzer2/FuncallSqlizer.scala

```diff
@@ -147,13 +163,45 @@ abstract class FuncallSqlizer[MT <: MetaTypes] extends SqlizerUniverse[MT] {
     }
   }
 
-  protected def sqlizeNormalOrdinaryFuncall(sqlFunctionName: String, prefixArgs: Seq[Doc] = Nil, suffixArgs: Seq[Doc] = Nil) = {
+  def sqlizeNormalOrdinaryFuncall(
+    sqlFunctionName: String,
+    prefixArgs: Seq[Doc] = Nil,
+    suffixArgs: Seq[Doc] = Nil,
+    castType: (Doc, Int) => Option[Doc] = (_, _) => None
```

Comment:
I'm not entirely sure I like this as-is - in particular, I think I'd prefer that if it needs to take that `Doc` parameter at all (from a quick glance I don't see anything that uses it) that it'd take an `ExprSql` and return an `ExprSql` instead of operating on `Doc`s.

Reply:
Ah, good point. Doc does not make sense here. I'll remove it since it's not used.